Make the guttering section more clear (#23937)
mdn/content: files/en-us/web/css/css_flexible_box_layout/mastering_wrapping_of_flex_items/index.md

## Creating gutters between items

> You may have heard great stories about negative margins. Don't even think. Nowadays we have the Flexbox `gap` property to make the said task super convenient.

Reviewer: ```suggestion
To create gaps or gutters between flex items, use the {{CSSXref('gap')}} property.
```

We can just get rid of the historical narrative.

The gap property in CSS is a shorthand for `row-gap` and `column-gap`, specifying the size of gutters, which is the space between rows and columns within grid, flex, and multi-column layouts.

> In Flexbox, the `gap` property is applied on the flex-container. It creates a fixed space between the flex-items but not on the sides. However, the gap property is not the only thing that can put space between items. Margins, paddings, justify-content and align-content can also increase the size of the gutter and affect the actual gap value. You should try out and see how the `gap` property differs from margin(in both axes) and how margin or padding affects it.

Reviewer: ```suggestion
In Flexbox, the `gap` property is applied to the flex container. It creates a fixed space between adjacent flex items. However, the `gap` property is not the only thing that can put space between items. Margins, paddings, `justify-content`, and `align-content` can also increase the size of the gutter, affecting the actual size of the gap. 

To see how the `gap` property differs from `margin` in both axes, try adding a `gap` to the container `.box` and change the `margin` value on the `.box > *` flex items.
```
